strr-base-web/app/composables/useStrrModals.ts

```diff
@@ -131,20 +131,29 @@ export const useStrrModals = () => {
    * @param {string} title - title of the modal
    * @param {string} content - content/description displayed in the modal
    * @param {string} confirmButtonLabel - label for the confirm button
+   * @param {string} cancelButtonLabel - label for the cancel button
    * @param {function} confirmHandler - function to be called when the confirm button is clicked
+   * @param {boolean} hideCancelButton - optional parameter to hide the cancel button
    */
   function openConfirmActionModal (
     title: string,
     content: string,
     confirmButtonLabel: string,
-    confirmHandler: () => void) {
+    cancelButtonLabel: string,
```

Comment:
There should be a default cancel button label, so we don't have to pass it all the time and only when need to override.

Reply:
I will address this in my next PR.